manifests/00-custom-resource-definition.yaml

```diff
@@ -442,22 +88,23 @@ spec:
                 hostNetwork:
                   description: hostNetwork holds parameters for the HostNetwork endpoint
                     publishing strategy. Present only if type is HostNetwork.
+                  nullable: true
```

Comment:
this makes the schema weaker. Probably you have a `// +kubebuilder:nullable=true` tag or so in the API types?

Reply:
Fixed in https://github.com/openshift/api/pull/438

Reply:
Fixed here.